velox/exec/trace/test/CMakeLists.txt

```diff
@@ -28,4 +28,11 @@ target_link_libraries(
   velox_memory
   velox_query_trace_exec
   velox_query_trace_retrieve
-  velox_vector_fuzzer)
+  velox_vector_fuzzer
+  GTest::gtest_main
+  GTest::gmock
+  Folly::folly
+  gflags::gflags
+  glog::glog
+  fmt::fmt
+  ${FILESYSTEM})
```

Comment:
Are you sure you need to explicitly link against all of these? I don't see any explicit `#includes` for any headers from these targets. The other targets will propagate their usage requirements, there is no need to add them manually.